src/sql/magic.py

```diff
@@ -436,6 +453,18 @@ def interactive_execute_wrapper(**kwargs):
                 if detailed_msg is not None:
                     err = exceptions.UsageError(detailed_msg)
                     raise err
+                    # TODO: move to error_messages.py
+                elif "no such table" in str(e):
+                    tables = query_util.extract_tables_from_query(command.sql)
+                    for table in tables:
+                        suggestions = get_close_matches(table, list(self._store))
+                        if len(suggestions) > 0:
+                            err_message = f"There is no table with name {table!r}."
+                            suggestions_message = get_suggestions_message(suggestions)
+                            raise exceptions.TableNotFoundError(
+                                f"{err_message}{suggestions_message}"
+                            )
+                    print(e)
```

Comment:
I was unable to get this working (ran the compose.md file):

![image](https://github.com/ploomber/jupysql/assets/989250/7acf2baf-7e12-4aa6-b088-2a7bcce4f282)


changed from genres_fav -> genres_fav2

Reply:
It would be `ON t.GenreId = genres_fav2.GenreId` (It's `genres_fav.GenreId` in the above query)

<img width="1104" alt="Screenshot 2023-05-26 at 1 20 53 PM" src="https://github.com/ploomber/jupysql/assets/29096135/7880ef1b-df23-4a8b-a07b-b00227b69603">

@edublancas